config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -249,7 +249,7 @@ presubmits:
     extra_refs:
     - org: jsturtevant # TODO(jstur): change it to 'kubernetes-sigs'
       repo: cluster-api-provider-azure
-      base_ref: windows-custom-binaries # TODO(jstur): change it to 'master'
+      base_ref: windows-custom-binaries # TODO(jstur): change it to 'main'
```

Comment:
```suggestion
      base_ref: main # TODO(jstur): change it to 'main'
```

Reply:
Should I revert the org to `kubernetes-sigs` as well? I'm assuming we want off the fork entirely.

Reply:
yes please!  It merged into capz so we are ready to cut over